src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -563,11 +587,36 @@ public void ProcessCommand(IExecutionContext context, string inputLine, ActionCo
             context.AddIssue(issue);
         }
 
+        static void ValidateLinesAndColumns(ActionCommand command) 
+        {
+            command.Properties.TryGetValue(IssueCommandProperties.Line, out string line);
+            command.Properties.TryGetValue(IssueCommandProperties.EndLine, out string endLine);
+            command.Properties.TryGetValue(IssueCommandProperties.Column, out string column);
+            command.Properties.TryGetValue(IssueCommandProperties.EndColumn, out string endColumn);
+
+            var hasColumnValue = column != null || endColumn != null;
+            var hasLine = line != null;
+            var hasEndLine = endLine != null;
+
+            if (!hasLine && hasColumnValue) 
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'column' and 'end_column' can only be set if 'line' value is provided."); 
+            }
+
```

Comment:
Should we also check if end_column is provided but column is not? Same with line. What happens if you post a check with an end-column but no start column?